libbeat/beat/beat.go

```diff
@@ -155,6 +156,10 @@ func (b *Beat) launch(bt Creator) error {
 		return err
 	}
 
+	if err := plugin.Initialize(); err != nil {
```

Comment:
The behaviour is the same. It's just that with a undocument config option, not even the initialize code would be execute. I would still prefer to have an additional config option :-)